skbio/alignment/_alignment.py

```diff
@@ -1405,6 +1408,84 @@ def sequence_length(self):
         else:
             return len(self._data[0])
 
+    def weblogo(self, reverse=False):
+        """Plots alignment as a weblogo.
+
```

Comment:
:+1:  more information would be great. I remember original paper uses information content rather than frequency in seq logo (?). would it be useful to provide optional parameter to calculate either of them?